tests/common/dualtor/dual_tor_utils.py

```diff
@@ -472,3 +473,30 @@ def shutdown(vm_names=None, upper=False, lower=False):
     logger.info('Recover T1 VM ports connected to tor')
     for eos_host, vm_intf in down_intfs:
         eos_host.no_shutdown(vm_intf)
+
+
+@pytest.fixture(scope='module', autouse=True)
+def start_linkmgrd_heartbeat(ptfadapter, duthost, tbinfo):
+    '''
+    Send a GARP with the PTF MAC to the DUT for each port configured with a mux cable
+
+    Linkmgrd will not start sending heartbeats until the PTF MAC is present in the DUT neighbor table
+    '''
+    garp_pkts = {}
+
+    ptf_indices = duthost.get_extended_minigraph_facts(tbinfo)["minigraph_ptf_indices"]
+    mux_cable_table = duthost.get_running_config_facts()['MUX_CABLE']
+
+    for vlan_intf, config in mux_cable_table.items():
+        ptf_port_index = ptf_indices[vlan_intf]
+        server_ip = ip_interface(config['server_ipv4'])
+        ptf_mac = ptfadapter.dataplane.ports[(0, ptf_port_index)].mac()
+
+        garp_pkt = testutils.simple_arp_packet(eth_src=ptf_mac,
```

Comment:
For GARP, don't you need to set `ip_tgt` too, same as the value of `ip_snd`?

Reply:
You are right, the default just happens to fall within the subnet of the default VLAN used for testing. Will update, thanks.